### ISSUE TITLE
Remove `aria-label` from `aria-hidden` icons

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#1427]
**Problematic behavior**
Icons hidden with the `aria-hidden="true"` attribute cannot have an `aria-label` attribute.

**Expected behavior/code**
`<span aria-hidden="true" aria-label="Émoticônes du document" class="sc-34780e96-0 sc-262648e5-0 iiwUsF ihSnKO">🇫🇷</span>`

**Steps to Reproduce**

1. Look for icons that are hidden with the `aria-hidden="true"` attribute in the left sidebar, and also in the document list when arriving on Docs.
2. Remove the `aria-label` attributes from these icons
3. and that's it!

<img width="950" height="363" alt="" src="https://github.com/user-attachments/assets/9343c80f-1dde-4479-998b-fb20cc156035" />
<img width="322" height="514" alt="" src="https://github.com/user-attachments/assets/7e1b22d9-0f1e-41c4-b5f3-58f1756c2eee" />

**Possible Solution**
Remove the `aria-label` attributes from icons hidden with `aria-hidden="true"`
